File: python/kisoen/kisoen/day2/fizzbuzz.py
#coding:utf-8
#FizzBuzz問題

print("(最後の)数字Nを入力してください")
n = int(input())

print('\n')
for i in range(1,n+1):
    if i % 3 == 0 and i % 5 == 0:
        print("FizzBuzz")
    elif i % 3 == 0:
        print("Fizz")
    elif i % 5 == 0:
        print("Buzz")
    else:
        print("{0}".format(i))

File: C/chap11-/fizzbuzz.c
#include<stdio.h>

int main(void)
{

  int i;
  int n;

  printf("input the last number>");
  scanf("%d", &n);

  for (i=1; i < n + 1; i++){
    if (i % 3 == 0 && i % 5 == 0){
      printf("fizz buzz\n");
    } else if (i % 3 == 0){
      printf("fizz\n");
    } else if (i % 5 == 0){
      printf("buzz\n");
    } else {
      printf("%d\n1", i);
    }
  }

  return 0;
}
